lib/hooks/ansi.nix

```diff
@@ -0,0 +1,13 @@
+# Ansi escape codes
+let
+  # Nix strings only support \t, \r and \n as escape codes, so actually store
+  # the literal escape "ESC" code.
+  inherit (builtins) foldl' genList;
+  esc = "";
```

Comment:
This should hold the literal escape code...

Looks like something in my auto ated pipeline has removed it. Should look like this: https://raw.githubusercontent.com/numtide/devshell/master/nix/ansi.nix

Reply:
No, all good, it's just that the GitHub app on the cell phone doesn't represent it right.